haec-user-service/src/modules/user/services/user.service.ts

```diff
@@ -56,4 +57,43 @@ export class UserService {
 
     return false;
   }
+
+// This will log the user ** IN **
+
+  public async loginUser(input: {
+    email: string;
+    password: string;
+  }): Promise<string> {
+    const verified = await this.verifyUser({ email: input.email, password: input.password})
+    if (verified == true){
+      // Generates random 21 character token with Crypto
```

Comment:
This seemed like a straightforward way to do it. There is a `danger of forward slash "/" causing problems if passed through the URL`, but we won't be passing a token through the url, so this should work fine.
`https://futurestud.io/tutorials/generate-a-random-string-in-node-js-or-javascript`